### DISCUSSION TITLE
MDR analysis

### DISCUSSION THREAD

**Discussion by ghost** [msberends/AMR#104] [Q&A]
Hello, i would first like to thank all the team working on AMR package, this is wonderful.

As I am quite new to coding in R, I would like to have some help. I am currently working with an antimicrobial resistance dataset, focusing on multi drug resistance patterns in methicillin resistant staphylococcus aureus (MRSA).

1)is it possible to differentiate and get the exact count between MDR, XDR and PDR, using the CMI2012 guidelines?

2)concerning the MRSA, termed as such when resistant (R) to cefoxitin (FOX), is it possible to pool the proportion of resistance of MRSA to other antibiotics (in other word this is to specifically answer the question: what proportion of MRSA are drug/multi drug resistant ?)

I am working with a standard dataset with column headers as: patient_id, service, bacteria, and the rest of the columns containing susceptibility/resistance data of antibiotics.

**Comment by msberends** [comment]
Oops, answered here: https://github.com/msberends/AMR/discussions/18#discussioncomment-5558675
